servicenow/6.11/modules/ROOT/pages/servicenow-connector-troubleshooting.adoc

```diff
@@ -0,0 +1,53 @@
+= Troubleshooting ServiceNow Connector 6.11 - Mule 4
+
+To troubleshoot Anypoint Connector for ServiceNow (ServiceNow Connector), become familiar with the information about interpreting commonly thrown exception messages.
+
+== Understand Commonly Thrown Exceptions
+
+Here is a list of commonly thrown exception messages and how to interpret them:
+
+* SERVICENOW:BAD_REQUEST
+
+ The request is incorrect, for example, there are invalid input headers or the service does not exist.
+
+* SERVICENOW:BAD_RESPONSE
```

Comment:
I would leave this one out, since I think it's probably not used at all - it's in the connector as it was present in the older version, but it has no use from what I checked.